Support/Scripts/prepare-android-toolchain.sh

```diff
@@ -11,7 +11,7 @@
 
 source "$(dirname "$0")/common.sh"
 
-[ $# -le 1 ] || die "usage: $0 [ARCH]"
+[ $# -le 2 ] || die "usage: $0 [ARCH] [PLATFORM]"
```

Comment:
We've been using `21` for quite some time. I think we should bump android
platform versions only when we need to. Something that builds for platform
version `21` will work with `23` as well, as far as I know.

On Mon, Aug 8, 2016 at 5:06 PM, Francis Ricci <notifications@github.com>
wrote:

> In Support/Scripts/prepare-android-toolchain.sh
> <https://github.com/facebook/ds2/pull/418#discussion_r73978851>:
>
> > @@ -11,7 +11,7 @@
> >
> >  source "$(dirname "$0")/common.sh"
> >
> > -[ $# -le 1 ] || die "usage: $0 [ARCH]"
> > +[ $# -le 2 ] || die "usage: $0 [ARCH] [PLATFORM]"
>
> We don't want to default to the most recent?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/ds2/pull/418/files/e248fd93289c97866b43137c53b6af07dfee8b19..891b41c62541fbda775166dfada5e546c9b3b8a0#r73978851>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AADHbKxzD1ZkAHImEpbF8e_vCxQoZ5ysks5qd8SSgaJpZM4JffU1>
> .
>



-- 
Stephane Sezer